RELEASE-NOTES.md

```diff
@@ -1,10 +1,22 @@
 # Prefect Release Notes
 
-## Release preview
+## Release 2.10.11
 
-- Adds work pool creation wizard — https://github.com/PrefectHQ/prefect/pull/9717
-- Add interactive prompts for deployment name and work pool to `prefect deploy` — https://github.com/PrefectHQ/prefect/pull/9707
-- Add interactive prompt to `prefect work-pool create` — https://github.com/PrefectHQ/prefect/pull/9686
+### Interactive Deployments and Work Pool Wizard 🧙
+![Screenshot 2023-05-25 at 1 02 48 PM](https://github.com/PrefectHQ/prefect/assets/12350579/c861b8dd-2dbb-4cfa-82f9-69008714f9fe)
+
+This release introduces enhancements aimed at simplifying deployment and work pool creation.
+
+Firstly, the `prefect deploy` command has been upgraded to provide interactive prompts for deployment names and work pool selections. If you don't provide a deployment name via the CLI or a deployment.yaml file, Prefect will prompt you to do so. Furthermore, if a work pool name isn't specified, Prefect will guide you through the available work pools for your workspace. This feature aims to make deployments more approachable, even for first-time users, requiring just an entrypoint to a flow to get started.
+
+Secondly, we've added a work pool creation wizard to streamline the process and spotlight various infrastructure types. The wizard will walk you through the essentials: basic work pool info, infrastructure type, and infrastructure configuration. The infrastructure type step will present you with a list of available infrastructure types, each with an icon and a description.
+
```

Comment:
A UI screenshot or two here would be super

Reply:
Added!